lib/series.js

```diff
@@ -0,0 +1,166 @@
+var assert = require('assert');
+var _ = require('lodash');
+
+
+/** Types supported for reporting */
+var types = {
+  String: function(v) { return /string/.test(typeof(v)); },
+  Number: function(v) { return /number/.test(typeof(v)); },
+  Any:    function(v) { return /string|number/.test(typeof(v)); }
+};
+
+// Export types
```

Comment:
not sure if a comment is really needed to define what's going on here.

Was wondering why this style of exporting is preferred over something like:

export const TYPES = {...}

Just for my own knowledge